Client/Frontend/Home/FxHomeLogoHeaderCell.swift

```diff
@@ -0,0 +1,49 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+fileprivate struct LogoViewUX {
+    static let imageHeight: CGFloat = 40
+    static let imageWidth: CGFloat = 214.74
```

Comment:
So this oddly precise width I guess must be the width of the provided asset?

Reply:
Lol, yes.